src/test/resources/com/puppycrawl/tools/checkstyle/checks/metrics/classfanoutcomplexity/InputClassFanOutComplexityAnnotations.java

```diff
@@ -0,0 +1,34 @@
+package com.puppycrawl.tools.checkstyle.checks.metrics.classfanoutcomplexity;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+public class InputClassFanOutComplexityAnnotations {
+
+    public void foo1(@TypeAnnotation char a) {}
+
+    public void foo2(final char @TypeAnnotation [] a){}
+
+    @MethodAnnotation
+    public void foo3(){}
+
+    @Override
+    public String toString() {
+        return super.toString();
+    }
+
+}
+
+@Target(ElementType.TYPE_USE)
+@interface TypeAnnotation {}
+
+@Target(ElementType.METHOD)
+@interface MethodAnnotation {}
+
+@MyAnnotation
+class MyClass {}
+
+@MyAnnotation
+interface MyInterface {}
```

Comment:
Please add multiline comment over top class to explain that such file supposed to be used with strict config.
Please add trailing comment on each line in input files where violation is expected

Reply:
What should i write in it?

Reply:
This input file is intended to be used on strict configuration: max=0

Reply:
added